src/shogun/modelselection/ParameterCombination.cpp

```diff
@@ -544,7 +543,7 @@ CDynamicObjectArray* CParameterCombination::non_value_tree_multiplication(
 	CDynamicObjectArray* result=new CDynamicObjectArray();
 
 	/* first step: get all names in the sets */
-	set<string> names;
+	std::set<std::string> names;
```

Comment:
is this change in parameter combination really necessary here?
This stuff is only every used in model selection code. I think it should be changed in a second patch